<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>194</width>
      <height>102</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton_start">
         <property name="text">
          <string>Timer Start!</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_pause">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_stop">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>STOP</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_timer_interval">
             <property name="text">
              <string>Timer Interval</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox_timer_interval_setup"/>
             </item>
             <item>
              <widget class="QLabel" name="label_show_second">
               <property name="text">
                <string>sec</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_show_count">
           <property name="text">
            <string>Count:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_show_count_num">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
